lib/src/object_id.rs

```diff
@@ -42,8 +42,16 @@ macro_rules! impl_id_type {
                 Self(bytes.to_vec())
             }
 
+            /// Parses the given hex string into an ObjectId.
+            ///
+            /// The given string is usually a literal, and must be valid.
```

Comment:
Do you think it would make sense to require a `'static` argument to reduce abuse?

Reply:
Yeah, I'll send a follow up.